rados/omap.go

```diff
@@ -105,6 +105,101 @@ func (ioctx *IOContext) SetOmap(oid string, pairs map[string][]byte) error {
 	return op.operateCompat(ioctx, oid)
 }
 
+// OmapKeyValue items are returned by the OmapGetElement's Next call.
+type OmapKeyValue struct {
+	Key   string
+	Value []byte
+}
+
+// OmapGetElement values are used to get the results of an GetOmapValues call
+// on a WriteOp. Until the Operate method of the WriteOp is called the Next
+// call will return an error. After Operate is called, the Next call will
+// return valid results.
+// The life cycle of the OmapGetElement is bound to the ReadOp, if the ReadOp
+// Release method is called the element may no longer be used.
+type OmapGetElement struct {
+	// inputs:
+	startAfter   string
+	filterPrefix string
+	maxReturn    uint64
+
+	// arguments:
+	cStartAfter   *C.char
+	cFilterPrefix *C.char
+
+	// C returned data:
+	iter C.rados_omap_iter_t
+	more C.uchar
+	rval C.int
+
+	// internal state:
+	iterable bool
```

Comment:
The usage if `iterable` is not obvious. Is this a sort of short-cut for `more != 0`? A clarifying comment would make it easier to understand.

Reply:
How about renaming it to `canIterate`?

Reply:
That does not make a real difference to me. A comment explaining how it is used/modified would be best.

Reply:
Renamed and comment added.